docs/guide/deployment/authorization-server.md

```diff
@@ -1,65 +1,168 @@
-| `-` | `SDK_OAUTH_AUDIENCE` | string |  `refer to the resource servers that should accept the token` | `""` |
-| `-` | `SDK_OAUTH_SCOPES` | string | `Comma separated list of required scopes` | `""` |
-| `-` | `SDK_OAUTH_RESPONSE_MODE` | string | `one of "query/post_form"`| `"query"` |
-| `-` | `LISTEN_TYPE` | string | `defines how to obtain listen TLS certificates - options: acme|file` | `"acme"` |
-| `-` | `LISTEN_ACME_CA_POOL` | string | `path to pem file of CAs` | `""` |
-| `-` | `LISTEN_ACME_DIRECTORY_URL` | string |  `url of acme directory` | `""` |
-| `-` | `LISTEN_ACME_DOMAINS` | string | `list of domains for which will be in certificate provided from acme` | `""` |
-| `-` | `LISTEN_ACME_REGISTRATION_EMAIL` | string | `registration email for acme` | `""` |
-| `-` | `LISTEN_ACME_TICK_FREQUENCY` | string | `interval of validate certificate` | `""` |
-| `-` | `LISTEN_ACME_USE_SYSTEM_CERTIFICATION_POOL` | bool | `load CAs from system` | `false` |
-| `-` | `LISTEN_FILE_CA_POOL` | string | `path to pem file of CAs` |  `""` |
-| `-` | `LISTEN_FILE_CERT_KEY_NAME` | string | `name of pem certificate key file` | `""` |
-| `-` | `LISTEN_FILE_CERT_DIR_PATH` | string | `path to directory which contains LISTEN_FILE_CERT_KEY_NAME and LISTEN_FILE_CERT_NAME` | `""` |
-| `-` | `LISTEN_FILE_CERT_NAME` | string | `name of pem certificate file` | `""` |
-| `-` | `LISTEN_FILE_USE_SYSTEM_CERTIFICATION_POOL` | bool | `load CAs from system` | `false` |
-| `-` | `LOG_ENABLE_DEBUG` | bool | `enable debugging message` | `false` |
-| `-` | `MONGODB_URI` | string | `uri to mongo database` | `"mongodb://localhost:27017"` |
-| `-` | `MONGODB_DATABASE` | string | `name of database` | `"authorization"` |
-| `-` | `LOG_ENABLE_DEBUG` | bool | `debug logging` | `false` |
+## YAML Configuration
+### Logging
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `log.debug` | bool | `Set to true if you would like to see extra information on logs.` | `false` |
+
+### gRPC API
+gRPC API of the Authorization Service as defined [here](https://github.com/plgd-dev/cloud/blob/v2/authorization/pb/service_grpc.pb.go#L19).
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.grpc.address` | string | `Listen specification <host>:<port> for grpc client connection.` | `"0.0.0.0:9100"` |
+| `api.grpc.tls.caPool` | string | `File path to the root certificate in PEM format.` |  `""` |
+| `api.grpc.tls.keyFile` | string | `File path to private key in PEM format.` | `""` |
+| `api.grpc.tls.certFile` | string | `File path to certificate in PEM format.` | `""` |
+| `api.grpc.tls.clientCertificateRequired` | bool | `If true, require client certificate.` | `true` |
+| `api.grpc.authorization.authority` | string | `Endpoint of oauth provider.` | `""` |
+| `api.grpc.authorization.audience` | string | `Audience of oauth provider API.` | `""` |
```

Comment:
Still not right :)
oauth to OAuth , but that's not where the main problem is.
"Identifier of the API configured in your OAuth provider."

Short explanation, you might have one client, but multiple APIs in the system. What you want to prevent is to be able to contact all the APIs of your system with one token. This audience allows you to request the token for a specific API. If you configure it to myplgdc2c.api in the Auth0, you have to set it here if you want to also validate it.

Reply:
Do I add "short explanation" in the configuration table ? or Other common place like bottom area of file ?